service/notesservice.php

```diff
@@ -196,6 +181,31 @@ public function delete ($id, $userId) {
         $file->delete();
     }
 
+    private function getSafeTitleFromContent($content) {
+        // prepare content: remove markdown characters and empty spaces
+        $content = preg_replace("/^\s*[*+-]\s+/m", "", $content); // list item
+        $content = preg_replace("/^#+\s+(.*?)\s*#*$/m", "$1", $content); // headline
```

Comment:
I wasn't aware of closing heading markup. I believe this currently leads to issues in the web editor. But this is unrelated to this PR.